.editorconfig

```diff
@@ -1,20 +1,21 @@
+
 root = true
 
 [*]
 charset = utf-8
 end_of_line = lf
 
+
 [*.java]
 indent_style = tab
-tab_width = 4
-trim_trailing_whitespace = true
 insert_final_newline = true
-ij_continuation_indent_size = 8
-max_line_length = 120
+trim_trailing_whitespace = true
+max_line_length = 150
```

Comment:
120 is already a lot (when diffing in the split mode). Google Java Style Guide says even 100: https://google.github.io/styleguide/javaguide.html#s4.4-column-limit

Reply:
I and KN are in favor of longer lines. Of course, some people prefer a more vertical code style., but one could always go lower than the limit.

Reply:
Screens are wide nowadays.

Reply:
My screen is not small either. However, I often work with the editor window split vertically (so I see two files side by side) and I view git diffs in the split mode. In this case, anything beyond 120 lines is not visible.

However, I do not want to be the blocker here. 150 is OK. Anything between 120 and 150 is even better ;-)